### DISCUSSION TITLE
Two questions regarding the listbox created from a duplicate search

### DISCUSSION THREAD

**Discussion by antodc** [doublecmd/doublecmd#1766] [General]
1) I would like to use a personalized view for that listbox (eg, add size and modification date to the 'standard' view), but I am not able to keep the 'group' column. What is the internal name of the 'group' field? There is a 'group' field the column personalization, but it gives another result.

2) In the 'Select a group' panel there is an option to 'Leave at least one file in each group unselected', which allows (among other things) to keep the largest/smallest file in each group. However, the listbox is the result of a duplicate search, so all files in a group have exactly the same size. Why should one ever select the largest/smallest file in each group?

**Comment by alexx2000** [comment]
1. It is a special column. It cannot be configured.

2. User can search a duplicates by name only, size can be different in this case.

**Reply by antodc** [reply]
Thanks a lot for the very quick answers!! I think it would be very useful if it were possible to make that column configurable.
